corehq/apps/app_manager/templates/app_manager/partials/form_session_endpoint.html

```diff
@@ -0,0 +1,44 @@
+{% load i18n %}
+{% load hq_shared_tags %}
+
+            <div class="form-group">
+                <label class="control-label {% css_label_class %}">
+                    {% trans 'Session Endpoint ID' %}
+                    <span class="hq-help-template"
+                        data-title="{% trans 'Session Endpoint ID' %}"
+                        data-content="{% blocktrans %}A session endpoint ID allows Android apps to call in to
```

Comment:
I take it this description is taken from somewhere else? If not I would change it to be more generic since we are using session endpoints in web apps as well.

Reply:
If there are no translations I would remove "android". Otherwise, just leave it.